src/screen/Drawer/DrawerScreen.tsx

```diff
@@ -20,126 +19,156 @@ import {
   parachainsOverviewScreen,
   parachainAuctionsScreen,
 } from 'src/navigation/routeKeys';
-import globalStyles, {monofontFamily, standardPadding} from 'src/styles';
+import {standardPadding} from 'src/styles';
 import {appVersion} from 'src/service/Device';
 import {getCurrentYear} from 'src/utils';
+import {Drawer, Switch, Text, Divider} from 'src/packages/base_components';
 
 function DrawerScreen({navigation}: DrawerContentComponentProps) {
   const {theme, toggleTheme} = useTheme();
   const isParachainAvailable = useIsParachainAvailable();
+  const [activeScreen, setActiveScreen] = React.useState('Dashboard');
 
   return (
     <SafeView>
-      <Layout style={styles.container}>
-        <View style={styles.main}>
-          <TouchableOpacity style={styles.logoContainer} onPress={() => navigation.navigate(dashboardScreen)}>
-            <Image source={logo} style={styles.logoImage} />
-            <Text style={styles.slogan}>Decentralized Identity</Text>
-          </TouchableOpacity>
-        </View>
-        <Layout style={styles.rest}>
-          <Divider />
-          <ListItem
-            title="Dashboard"
-            accessoryLeft={(props) => <Icon {...props} name="browser-outline" animation="zoom" />}
-            onPress={() => navigation.navigate(dashboardScreen)}
-          />
-          <Divider />
-          <ListItem
-            title="Accounts"
-            accessoryLeft={(props) => <Icon {...props} name="person-outline" animation="zoom" />}
-            onPress={() => navigation.navigate(accountsNavigator)}
+      <View>
+        <TouchableOpacity style={styles.logoContainer} onPress={() => navigation.navigate(dashboardScreen)}>
+          <Image source={logo} style={styles.logoImage} />
+          <Text style={styles.slogan}>Decentralized Identity</Text>
+        </TouchableOpacity>
+      </View>
+      <Divider />
+      <ScrollView>
+        <Drawer.Section>
+          <Drawer.Item
+            label="Dashboard"
+            icon="view-dashboard"
+            active={activeScreen === 'Dashboard'}
+            onPress={() => {
+              setActiveScreen('Dashboard');
```

Comment:
```suggestion
            active={activeScreen === dashboardScreen}
            onPress={() => {
              setActiveScreen(dashboardScreen);
```
also for other routes